app/services/service_result.rb

```diff
@@ -162,21 +207,45 @@ def add_dependent!(dependent)
     self.dependent_results += inner_results
   end
 
+  ##
+  # Execute block argument if the Service call succeeded.
+  #
+  # @yield block to be called on success.
+  # @return [self]
   def on_success(&)
     tap(&) if success?
     self
   end
 
+  ##
+  # Execute block argument if the Service call failed.
+  #
+  # @yield block to be called on failure.
+  # @return [self]
   def on_failure(&)
     tap(&) if failure?
     self
   end
 
+  ##
+  # Iterates exactly once, passing the result to the block, if the Service call succeeds.
```

Comment:
```suggestion
  # Iterates exactly once, passing the result to the block, if the Service call succeeded.
```

Past tense is already used in the above methods, so let's keep it consistent.

Reply:
I've fixed the tense for all of the param docs.